graspologic/inference/lpt.py

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+
+np.random.seed(88888889)
```

Comment:
that's weird in a source file. importing a file shouldn't mess up someone's rng seed.